### ISSUE TITLE
`Bun.serve` provides incorrect status text "HM"

### ISSUE THREAD

**Issue by paperclover** [oven-sh/bun#1520]
### What version of Bun is running?

0.2.3 canary

### What platform is your computer?

Linux 6.0.7-arch1-1 #1 SMP PREEMPT_DYNAMIC Thu, 03 Nov 2022 18:01:58 +0000 x86_64 unknown

### What steps can reproduce the bug?

- start an http server with anything and `fetch()` or `curl` it

Bun test case, needs `curl` installed
```ts
import { expect, test } from 'bun:test';

test('response status is OK', async () => {
  const server = Bun.serve({
    fetch() {
      return new Response('Hello World');
    },
    port: 43032,
  });
  const process = Bun.spawn(['curl', '-I', 'http://localhost:43032']);
  const response = (await new Response(process.stdout).text()).split('\n')[0];
  server.stop();
  expect(response).toBe('HTTP/1.1 200 OK');
});
```

### What do you see instead?

HTTP/1.1 200 HM
HTTP/1.1 404 HM
etc

### Additional information

related to #866 but this is reporting that the built in status texts do not function.

**Comment by Jarred-Sumner** [comment]
statusText is deprecated in HTTP/2 

I agree "HM" is a weird choice

**Comment by Electroid** [comment]
Yea we should just confirm to what the defaults are.

**Comment by Electroid** [comment]
This has been fixed in [Bun v0.3.0](https://bun.sh/blog/bun-v0.3.0) 🎉
